developer-workflow/psrt.rst

```diff
@@ -0,0 +1,148 @@
+
+* Author the vulnerability advisory text. The advisory must include the following information:
+
+  * Title should be a brief description of the vulnerability and affected component (e.g "Buffer over-read in SSLContext.set_npn_protocols()")
+
+  * Short description of the vulnerability, impact, and the conditions where the affected component is vulnerable, if applicable.
+
+  * Affected versions. This could be "all versions", but if the vulnerability exists in a new feature
+    or removed feature then this could be different. Include versions that are end-of-life in this calculation.
+    (e.g. "Python 3.9 and earlier", "Python 3.10 and later", "all versions of Python")
+
+  * Affected components and APIs. The module, function, class, or method must be specified so users can
+    search their codebase for usage. For issues affecting the entire project, this can be omitted.
+
+  * Mitigations for the vulnerability beyond upgrading to a patched version, if applicable.
+
+  This can all be done within the GitHub Security Advisory UI for easier collaboration between reporter and coordinator.
+
+* Determine the fix approach and who will provide a patch. Some reporters are willing to provide or collaborate to create a
+  patch, otherwise relevant core developers can be invited to collaborate.
+
+* For **Low** and **Medium** severity vulnerabilities it is acceptable to develop a patch in public.
+  The pull request must be marked with the ``security`` and ``release-blocker`` labels so that a release
+  is not created without including the patch.
+
+* For **High** and **Critical** severity vulnerabilities the patch must be developed privately using GitHub Security Advisories'
+  "Private Forks" feature. Core developers can be added to the GitHub Security Advisory via "collaborators" to work
+  on the fix together. Once a patch is approved privately and tested, a public issue and pull request can be created
+  with the ``security`` and ``release-blocker`` labels.
+
+* Once the pull request is merged the advisory can be published. Send an advisory email to ``security-announce@python.org``
+  using the below template. Backport labels must be added as appropriate. After the advisory is published a CVE record
+  can be created.
+
+Template responses
+~~~~~~~~~~~~~~~~~~
+
+These template responses should be used as guidance for messaging
+in various points in the process above. They're not required to be sent as-is,
+please feel free adapt them as needed for the current context.
+
+**Directing to GitHub Security Advisories:**
+
+ .. code-block::
+
+   Thanks for submitting this report.
+   We now use GitHub Security Advisories for triaging vulnerability reports,
+   are you able to submit your report directly to GitHub?
+
+   https://github.com/python/cpython/security/advisories/new
+
+   If you're unable to submit a report to GitHub (due to not having a GitHub account or something else)
```

Comment:
It would improve usability for admins to limit the line lengths of the templates to fewer than 80 columns or so.  And, since these are intended to be somewhat formal responses and to people who may not have English as a first language, it might be better to avoid the use of contractions (`you're`, `I'll`, etc) throughout the templates.

Reply:
Resolved in https://github.com/python/devguide/pull/1348/commits/0344b7ce0b59c9ecebb41e28f353ed67ab704b30